Dockerfile

```diff
@@ -4,13 +4,21 @@ ENV JASMINEGRAPH_HOME="/home/ubuntu/software/jasminegraph"
 
 WORKDIR /home/ubuntu/software/jasminegraph
 
+RUN apt-get update
+RUN apt-get install -y python3.8
+RUN apt-get install -y python3-pip
+RUN apt-get install -y python3.8-distutils
+RUN python3.8 -m pip install stellargraph
+RUN python3.8 -m pip install chardet
+
 COPY ./GraphSAGE ./GraphSAGE
 RUN pip install -r ./GraphSAGE/requirements
+RUN pip install pandas
 
 COPY ./build.sh ./build.sh
 COPY ./run-docker.sh ./run-docker.sh
 COPY ./CMakeLists.txt ./CMakeLists.txt
-COPY ./src_python ./src_python
+COPY ./src_python ./src-python
```

Comment:
Why do we need to change the folder name from `src_python` to `src-python` ?

Reply:
In all other places in the repository has referred the folder as this
Check:
https://github.com/miyurud/jasminegraph/blob/master/conf/jasminegraph-server.properties#L59C2-L59C2

This can be done other way around as well.

Reply:
The following two locations we will be removing from the source path,

```
org.jasminegraph.fl.dataDir=/home/ubuntu/software/jasminegraph/src-python/data/
org.jasminegraph.fl.weights=/home/ubuntu/software/jasminegraph/src-python/weights/
```
I think the reason why do the above change it to map with the following,

```
org.jasminegraph.fl.location=/home/ubuntu/software/jasminegraph/src-python/
```

Therefore instead can we do the following change,

```
org.jasminegraph.fl.location=/home/ubuntu/software/jasminegraph/src_python/
```

and then keep the following as it is,

```
COPY ./src_python ./src_python
```

Reply:
Done

Reply:
Ok